### ISSUE TITLE
Create OME-Zarr structure Task failure when metadata does not match acquisition

### ISSUE THREAD

**Issue by nrepina** [fractal-analytics-platform/fractal-tasks-core#627]
Task fails when Yokogawa mrl metadata contains channels of different pixel dimensions (e.g. due to different binning settings). See example metadata; channel 5 was not used during acquisition but it is still in metadata due to inheritance from a previous acquisition, and causes the task to fail. Perhaps best to check consistency only on channels used for creating ome-zarr structure. 

```
ValueError: During metadata parsing, a consistency check failed: 
Image dimensions
Difference dataframe: 
horiz_pixel_dim       0.325
vert_pixel_dim        0.325
horiz_pixels       1000.000
vert_pixels        1000.000
bit_depth             0.000
dtype: float64

```
[MeasurementDetail.txt](https://github.com/fractal-analytics-platform/fractal-tasks-core/files/13565817/MeasurementDetail.txt)
[MeasurementData.txt](https://github.com/fractal-analytics-platform/fractal-tasks-core/files/13565818/MeasurementData.txt)

**Comment by jluethi** [comment]
Agreed, we should refactor the consistency check to only run for channels that are actually present. Either based on allowed_channels or based on channels present in the mlf file.

**Comment by jluethi** [comment]
For future reference: See also additional description in https://github.com/fractal-analytics-platform/fractal-tasks-core/issues/287
